planning/behavior_path_planner/src/path_utilities.cpp

```diff
@@ -66,7 +67,7 @@ PathWithLaneId resamplePathWithSpline(const PathWithLaneId & path, double interv
     transformed_path.at(i) = path.points.at(i).point;
   }
 
-  constexpr double epsilon = 0.01;
+  constexpr double epsilon = 0.1;
```

Comment:
0.01 seems too close and sometimes causes path distortion.

Reply:
```suggestion
  constexpr double epsilon = 0.2;
```

is better from @TakaHoribe